testsuite/integration/basic/pom.xml

```diff
@@ -731,7 +731,7 @@
                                         <exclude>org/jboss/as/test/integration/jca/bootstrap/**/*TestCase*.java</exclude>
                                         <exclude>org/jboss/as/test/integration/jmx/full/**/*TestCase*.java</exclude>
                                         <exclude>org/jboss/as/test/integration/messaging/**/*TestCase*.java</exclude>
-                                        <include>org/jboss/as/test/integration/microprofile/opentracing/**/*TestCase*.java</include>
+                                        <exclude>org/jboss/as/test/integration/ejb/**/jts/**/*TestCase*.java</exclude>
                                         <exclude>org/jboss/as/test/integration/management/cli/**/*TestCase.java</exclude>
```

Comment:
Another possibility is, `<include>` might be a typo; it should be `<exclude>`. If that's the case, the fix is simply replacing `in` with `ex`.

The test `org/jboss/as/test/integration/ejb/**/jts/**/*TestCase*.java` is already excluded here. So this change will simply add a duplicate.